### PR TITLE
Completed all odoo module exercises

### PR DESCRIPTION
**Pull request by antonag32**
All exercises to be implemented in the Odoo module have been completed. Only exercise missing is to implement the XML-RPC application, said exercise will be posted on a different PR.

### CODE REVIEW

**Comment by frahikLV** [comment]
@antonag32 can you rebase this PR?

**Comment by antonag32** [comment]
> @antonag32 can you rebase this PR?

@frahikLV Just rebased it, thanks.

File: opa/i18n/en.po [hunk 1]
```diff
@@ -0,0 +1,314 @@
+# Translation of Odoo Server.
```
Comment (frahikLV) [inline opa/i18n/en.po]: Remove this file, it's redundant

File: opa/i18n/opa.pot [hunk 2]
```diff
@@ -0,0 +1,311 @@
+# Translation of Odoo Server.
```
Comment (frahikLV) [inline opa/i18n/opa.pot]: remove this file too

File: opa/views/session_views.xml [hunk 3]
```diff
@@ -130,4 +130,49 @@
...
+
+    <record id="action_graph_session" model="ir.actions.act_window">
+        <field name="name">Attendees by Course</field>
+        <field name="res_model">session</field>
+        <field name="view_mode">graph</field>
+        <field name="view_id" ref="session_graph_view"/>
+    </record>
+
+    <record id="action_calendar_session" model="ir.actions.act_window">
+        <field name="name">Sessions' Calendars</field>
```
Comment (frahikLV) [inline opa/views/session_views.xml]: "Schedule of sessions" or "Calendar of Sessions"

**Review by frahikLV — CHANGES_REQUESTED** [review]
@antonag32 can you address these changes?

File: opa/i18n/en.po (hunk 1 above)
Comment (antonag32) [reply opa/i18n/en.po]: Yes, I will remove it.

File: opa/i18n/opa.pot (hunk 2 above)
Comment (antonag32) [reply opa/i18n/opa.pot]: Vauxoo guidelines on the technical on-boarding file said we should version the .pot file too and I see all odoo modules do the same too. Do I still remove it?

File: opa/views/session_views.xml (hunk 3 above)
Comment (antonag32) [reply opa/views/session_views.xml]: I'll change it.

**Comment by antonag32** [comment]
@frahikLV Thanks for the review, I will attend the requests however I am confused about keeping or not the .pot file.

**Comment by frahikLV** [comment]
@antonag32 remove the .pot file, for client modules we don't use it

**Comment by antonag32** [comment]
> @antonag32 remove the .pot file, for client modules we don't use it

@frahikLV  Ok, all changes have been done. Thanks.

**Review by frahikLV — APPROVED** [review]
@antonag32 I will merge it,

**Pull request merged**